src/components/Markdown.jsx

```diff
@@ -1,14 +1,55 @@
 //
-// This file only contains exported styled components. These components are used
-// to replace the default components in the "markdown-to-jsx" package. See
-// the `[...path_params].js` file to see how they are used.
+// This file only contains exported styled components. These components are
+// used to replace the default components in our markdown package that we use
+// in `[...path_params].jsx`.
 //
+import React from "react";
+
```

Comment:
Noticed this is declared in 2 places (same as the function below), any way we can declare them one and re use?

Reply:
woops! i originally had it in the path_params component just to test it out and forgot to take it out after i moved it to the Markdown component. done! 83ccdd7